bin/travis/before_install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+sudo install -o "$USER" -d /etc/blocktogether
+sudo install -o "$USER" -d /data/blocktogether
+cp -a "$(dirname ${BASH_SOURCE[0]})/../../"config/* /etc/blocktogether/
```

Comment:
For this type of stuff, I usually put a `cd $(dirname $0)` at the top of the file. You could do the same with `${BASH_SOURCE[0]}` (I assume that's more broadly accurate?).